pkg/k8s/dto.go

```diff
@@ -21,3 +28,42 @@ func DefaultResources(res *sharedAPI.Resources) {
 	res.CPU = "1"
 	res.Memory = "1Gi"
 }
+
+// CheckDeploymentResourcesChanged checks if any of the deployment resources passed by the users have changes in shareAPI.Resources
+func CheckDeploymentResourcesChanged(res *sharedAPI.Resources) bool {
+	if res.CPU != "" ||
+		res.CPULimit != "" ||
+		res.Memory != "" ||
+		res.MemoryLimit != "" ||
+		res.Storage != "" {
+		return true
```

Comment:
Check only for `cpu` and `memory` changes.
All other properties won't affect pod rescheduling to a k8s node.